Smartling.API/Job/JobApiClient.cs

```diff
@@ -78,35 +78,37 @@ public virtual AddFileToJobResponse AddLocale(string localeId, string jobId, boo
       return JsonConvert.DeserializeObject<AddFileToJobResponse>(response["response"]["data"].ToString());
     }
 
-    public virtual List<Smartling.Api.Model.Job> Get(string jobName = "")
+    public virtual List<Smartling.Api.Model.Job> Get(string jobName = "", string statusFilter = "")
     {
-      var page = GetPage(jobName, PageSize, 0);
+      var page = GetPage(jobName, PageSize, 0, statusFilter);
       var results = new List<Smartling.Api.Model.Job>();
       var pageNumber = 0;
       results.AddRange(page.items);
 
       while (page.totalCount > results.Count && page.items.Count > 0)
       {
         pageNumber++;
-        page = GetPage(jobName, PageSize, PageSize * pageNumber);
+        page = GetPage(jobName, PageSize, PageSize * pageNumber, statusFilter);
         results.AddRange(page.items);
       }
 
       return results;
     }
-
-    public virtual JobList GetPage(string jobName, int limit, int offset)
+    
+    public virtual JobList GetPage(string jobName, int limit, int offset, string statusFilter = "")
     {
-      StringBuilder uriBuilder;
-      if (string.IsNullOrEmpty(jobName))
+      var url = string.Format(GetJobUrl, projectId, limit, offset);
+      if (!string.IsNullOrEmpty(jobName))
       {
-        uriBuilder = this.GetRequestStringBuilder(string.Format(GetJobUrl, projectId, limit, offset));
+        url += "&jobName=" + jobName;
       }
-      else
+
+      if (!string.IsNullOrEmpty(statusFilter))
       {
-        uriBuilder = this.GetRequestStringBuilder(string.Format(GetJobByNameUrl, projectId, jobName, limit, offset));
+        url += "&translationJobStatus=" + statusFilter;
```

Comment:
Now we can't guess what developer will pass here, so remove `translationJobStatus=`.
I worry that we can't properly urlencode `statusFilter` too.

Reply:
Maybe parse `statusFilter` on by key-value and concat again with encoding?

Reply:
@dimitrystd Maybe pass allowed statuses as string array?

Reply:
@dimitrystd Please see most recent commit with example